tools/check-links

```diff
@@ -97,11 +97,11 @@ class Crawler:
             hrefs.extend(script_srcs)
         return hrefs
 
-    def in_allowed_file_soup(self, url) -> bool:
-        if url[-1] == "/":
+    def in_allowed_file_soup(self, url_response) -> bool:
+        if url_response.url[-1] == "/":
             return True
```

Comment:
This check isn't needed, right? Don't requests to `localhost/benchmarks/` return `content-type: text/html` already?